database/database.js

```diff
@@ -114,14 +114,12 @@ module.exports = {
 	AddSourceValue : function(sourceValueJSON, collectionName) {
 		var SourceValueModel = require('../models/SourceValues.js')(collectionName);
 		
-
-			// console.log("Trying to add source value: " + sourceValueJSON + " into collection: " + collectionName);
 		try {
 			var createdSourceValue = new SourceValueModel(sourceValueJSON);
 		
 			createdSourceValue.save().then(function (res) {
 
-				// console.log("Successfully added source value " + res + " into collection " + collectionName);
+			//console.log("Successfully added source value " + res + " into collection " + collectionName);
```

Comment:
Why is this here? If it is not needed remove console.log and maybe add a comment instead.